packages/select/select.scss

```diff
@@ -277,11 +277,17 @@ $_select-search-input-selection-color--inverted: color.scale(
         padding: var(--jkl-select-option-padding);
         cursor: pointer;
 
-        &:focus,
-        &:hover {
```

Comment:
Trenger vi fjerne `hover` her? Problemet oppstod vel fordi noe _annet_ enn det som ble hovret over hadde fokus, og det har vi ikke lenger med React-endringen. 

Denne endringen vil brekke hovereffekten for de som ikke måtte bruke React-komponenten.

Reply:
Jepp, du har helt rett! Skal putte det tilbake. Bra catcha

Reply:
Hmm.. Når jeg gjør dette får vi en oppførsel der flere valg er markert når man bytter valg med piltastene samtidig som musepekeren er over et av valgene. Det skaper jo litt samme problem som vi forsøker å fikse i utgangspunktet 😅 Må gå litt tilbake i tenkeboksen her og se om vi kan få i både pose og sekk med dette.

Reply:
OK, du kommer sikkert til å hate denne løsningen @wkillerud, men det funker 😅 Jeg slang på et data-flagg på lista med valg i React-komponenten og markerer bare valget ved hover hvis _ikke_ det flagget er satt. Da kan jo også de som bare bruker stilpakken velge å sette det flagget dersom de har sin egen versjon av fokushåndteringen.